backend/embedding/main.py

```diff
@@ -48,8 +53,12 @@ def get_exec_id() -> str:
     return task_id
 
 
+@retry(tries=4, delay=2)
```

Comment:
Occasionally, It has experienced that access to RDS has failed. 
When this error occurred, the synchronization status was not updated on the database. 
A retry mechanism was added